jep/200/README.adoc

```diff
@@ -384,11 +384,18 @@ N/A
 
 Current state:
 
-* link:https://github.com/jenkinsci/acceptance-test-harness[Jenkins Acceptance Test Harness] has been executed with the patched components, several plugins were fixed (see the _Testing_ section)
-* link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compatibility Tester (PCT)] has been executed for a limited plugin set
+* link:https://github.com/jenkinsci/acceptance-test-harness[Jenkins Acceptance Test Harness (ATH)] has been executed with the patched components, several plugins were fixed (see the _Testing_ section)
+** After the discoveries in PCT (see below), the ATH results need to be reviewed carefully
+** BDFL Delegate is waiting for info about what has been tested
+(Jenkins WAR from link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120] or a custom build with custom plugin versions)
+* link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compatibility Tester (PCT)] has been executed for a limited plugin set for a sponsor's custom Jenkins WAR
 ** Not all plugins recommended in the Setup Wizard have been verified
+** The verified plugins have been checked with the sponsor's special set of plugins, not with the latest releases of plugins in Jenkins update centers.
+It cannot be considered as a reliable check since newer versions may contain other classes being serialized over Remoting/XStream.
```

Comment:
Possible of course, but unlikely. The plugins are pretty close to current, and the problematic classes _tend_ to be ones which have been in use since “old times”.